stl/inc/utility

```diff
@@ -134,6 +134,16 @@ _EXPORT_STD struct piecewise_construct_t { // tag type for pair tuple arguments
 
 _EXPORT_STD _INLINE_VAR constexpr piecewise_construct_t piecewise_construct{};
 
+_EXPORT_STD  struct _Ignore { // struct that ignores assignments

```

Comment:
`_EXPORT_STD` causes the name `_Ignore` to be user-visible when using `import std;`, which is undesirable.

In general, `_EXPORT_STD` should only be applied to names specified in the standard (and are not exposition-only), unless it's needed to work around a compiler bug.

```suggestion
struct _Ignore { // struct that ignores assignments
```

Reply:
Hmm i missed seeing this, should I implement this as well?

Reply:
Yes, this is important.

Reply:
Sure will do